.github/workflows/sync-onnxrt-main.yaml

```diff
@@ -0,0 +1,45 @@
+name: Onnxruntime main weekly sync
+on:
+  schedule:
+    - cron: "00 01 * * 4"
+
+jobs:
+  sync:
+    steps:
+      - uses: actions/checkout@v3
+        with:
+        ref: develop
+        path: ../
+
+  get_date:
+    steps:
+      - run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+  update_file:
+    needs: [sync get_date]
+    steps:
+      - run: git clone https://github.com/microsoft/onnxruntime.git && cd onnxruntime && git rev-parse HEAD >> ../test/onnx/.onnxrt-commit
+
+  Add_commit:
+    needs: update_file
+    steps:
+      - name: Add & Commit
+        uses: EndBug/add-and-commit@v9.1.1
+        with:
+          new_branch: onnxruntime-sync-${{ steps.date.outputs.date }}
+          add: ../test/onnx/.onnxrt-commit
+          message: Update Onnxruntime commit to latest release
+          default_author: github_actions
+          push: true
+  PR:
+    needs: Add_commit
+    steps:
+      - name: GitHub Action for creating Pull Requests
+        uses: devops-infra/action-pull-request@v0.5.3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          title: Sync Onnxruntime main
+          reviewer: TedThemistokleous, pfultz2, causten
+          assignee: TedThemistokleous, pfultz2, causten
```

Comment:
If you need to touch the yaml again, just make you the assignee, and then pfultz2 & causten as the reviewers

Reply:
Will do. What's odd though is I'm still not seeing this trigger even though I've adjusted things for UTC. Hm..My concern right now is suddenly waking up Friday and we have a bunch of PRs...

Reply:
Actually just found this:

"Scheduled workflows run on the latest commit on the default or base branch."

https://stackoverflow.com/questions/63436541/github-action-workflow-schedule-not-working-on-non-default-branch

Looks like I should be targeting this with master instead of develop with cron